scripts/templates/webdriverio.tpl.d.ts

```diff
@@ -16,6 +16,8 @@ declare namespace WebdriverIO {
         height: number
     }
 
+    interface RectReturn extends WebDriver.RectReturn {}
```

Comment:
Why do we need to declare it here? Can we just use WebDriver.RectReturn in command?

Reply:
Yes, you are right, I will remove it